nrf_dm/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,51 @@ Changelog
 
 All the notable changes to this project are documented in this file.
 
+main
+****
+
+All the notable changes added to the |NCS| main are documented in this section.
+
+Added
+=====
+
+* Added experimental support for the nRF5340 network core:
+
+  * ``lib/nRF5340_CPUNET/soft_float/libnrf_dm.a``
+
+* Added experimental support for a reduced library for the nRF5340 application core:
+
+  * ``lib/nRF5340_CPUAPP/hard_float/libnrf_dm_calc.a``
```

Comment:
Do we mention somewhere which functions are implemented by this lib?

Reply:
Actually I inspected the library and I only find that it provides `nrf_dm_calc`, not `nrf_dm_high_precision_calc` or `nrf_dm_populate_report` which I expected?

Reply:
I guess also `nrf_dm_version_string_get` would make sense